bot.js

```diff
@@ -119,8 +119,8 @@ const Client = require('facebook-messenger-puppeteer')
     if (/^i'?m /gi.test(message.body)) {
       if (ignore[message.thread] !== true) {
         ignore[message.thread] = timerIgnore()
-        const str = message.body.split('.')
-        str = str[0].replace(/i'?m/gi, "Hello")
+        const str = message.body.split(' ')
+        str = str.replace(/[&\/\\#,+()$~%.'":*?<>{}!]/g, '').replace(/i'?m/gi, "Hello ")
```

Comment:
* Might be worthwhile to also regex replace the **`** character
* Also, should only the first instance of `i(')m` be replaced (at the start as well?)